scripts/core/EXT_Template.rst

```diff
@@ -0,0 +1,42 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+:orphan:
+
+==================================
+Unified Runtime Extension Template
+==================================
+
```

Comment:
Couple of ideas for sections in this template:
1) Do we want a section to list contributors?
2) Should we specify the version of the UR spec the extension was written against?